app/controllers/partners/family_requests_controller.rb

```diff
@@ -14,10 +14,16 @@ def new
     end
 
     def create
-      # The checkbox toggles in the frontend will directly set active/inactive children,
-      # so fetching them from the DB here instead of reading them from params works.
-      # However this is a bit of an odd pattern so we should consider taking in params as usual here
-      children = current_partner.children.active.where.not(item_needed_diaperid: [nil, 0])
+      children_ids = []
+
+      params.each do |key, _|
+        is_child, id = key.split('-')
+        if is_child == 'child'
```

Comment:
@Beigelman are there chances that the params wouldn't be a child?

Reply:
I used this strategy to catch all the children ids because there are other fields inside the params hash such as "authenticity_token" or "action"